rtloader/test/rtloader/rtloader.go

```diff
@@ -345,3 +345,35 @@ func setModuleAttrString(module string, attr string, value string) {
 	C.release_gil(rtloader, state)
 	runtime.UnlockOSThread()
 }
+
+func getFakeModuleWithBool() (bool, error) {
+	var module *C.rtloader_pyobject_t
+	var class *C.rtloader_pyobject_t
+	var value C.bool
+
+	runtime.LockOSThread()
+	state := C.ensure_gil(rtloader)
+
+	// class
+	moduleStr := (*C.char)(helpers.TrackedCString("fake_check"))
+	defer C._free(unsafe.Pointer(moduleStr))
+
+	// attribute
+	attributeStr := (*C.char)(helpers.TrackedCString("foo"))
+	defer C._free(unsafe.Pointer(attributeStr))
+
+	ret := C.get_class(rtloader, moduleStr, &module, &class)
+	if ret != 1 || module == nil || class == nil {
+		return false, fmt.Errorf(C.GoString(C.get_error(rtloader)))
+	}
+
+	ret = C.get_attr_bool(rtloader, module, attributeStr, &value)
+	if ret != 1 {
+		return false, fmt.Errorf(C.GoString(C.get_error(rtloader)))
+	}
+
+	C.release_gil(rtloader, state)
+	runtime.UnlockOSThread()
+
+	return value == C.bool(true), nil
```

Comment:
maybe:
```suggestion
	return bool(value), nil
```
?

not sure if that's makes any different though 🤔

Reply:
971662fda75a780f9863293d106c00e732b8a84c

Reply:
this is not working I think